src/AppBundle/Tests/Controller/DefaultControllerTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AppBundle\Tests\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+/**
+ * Functional test that implements a "smoke test" of all the public and secure
+ * URLs of the application.
+ * See http://symfony.com/doc/current/best_practices/tests.html#functional-tests.
+ *
+ * Execute the application tests using this command (requires PHPUnit to be installed):
+ *
+ *     $ cd your-symfony-project/
+ *     $ phpunit -c app
+ *
+ */
+class DefaultControllerTest extends WebTestCase
+{
+    /**
```

Comment:
__[development][code_sniffer]__
* Doc comment for parameter "$url" missing

<!-- Comment by [SideCI](https://sideci.com) -->